src/test/java/com/google/ehub/servlets/LoginServletTest.java

```diff
@@ -29,42 +28,35 @@
 import org.junit.runners.JUnit4;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
- 
+
 @RunWith(JUnit4.class)
 public class LoginServletTest {
-  private static final String JSON_CONTENT_TYPE = "application/json";
-  private static  String LOGOUT_URL = "/logout";
-  private static  String LOGIN_URL = "/login";
-  private static final String userId = "12345";
-  private static final String email = "abc@gmail.com";
-  private static final String authDomain = "gmail.com";
-  private static final User user= new User(email, authDomain , userId);
-  private static final String ERROR_MESSAGE = "Error Message"; 
- 
+  private static String LOGOUT_URL;
+  private static String LOGIN_URL;
```

Comment:
You can delete these fields and make them local variables inside the test cases.